config/connect-log4j.properties

```diff
@@ -18,6 +18,19 @@ log4j.rootLogger=INFO, stdout
 
 log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
+
+#
+# The layout of each log message can be modified to include the Mapped Diagnostic Context parameters used by Connect,
+# which will include connector-specific and task-specific information on each log message and making it easier to identify
+# which log messages belong to which connector and tasks.
+#
+# The following parameters specified in the layout will be replaced with the corresponding values in every log message:
+#
+# Use %X{connector.name} for the name of the connector
+# Use %X{connector.scope} for the scope within a connector (e.g., "worker", "task", or "offsets")
+# Use %X{connector.task} for the 1-based number of the task, or blank if not within a connector task
+#
+#log4j.appender.stdout.layout.ConversionPattern=[%d] %p %X{connector.name}|%X{connector.scope}%X{connector.task} %m (%c:%L)%n
```

Comment:
Should we enable this by default in the provided `connect-log4j.properties`?

Reply:
log format is a public interface, if we do something like this we'd need to consider the compatibility implications. (we're already probably more lax in log message changes than we really should be...)

Reply:
Ok, that's why I had only included it as a comment.